### PR TITLE
Configuration YAML Documentation Updates

### PR DESCRIPTION
**Pull request by luis-camero**
- **live.mdx**:
  - documented changes to move `setup_path` to other directories than `/etc/clearpath`

- **yaml.mdx**:
  - removed `micro_ros_ws` that is no longer required
  - fixed issues with the platform `ros_parameters`
  - documented how to change between `ps4` and `logitech` 

- **controllers.mdx**:
  - add link to **yaml.mdx**

### CODE REVIEW

File: docs/ros/config/yaml.mdx [hunk 1]
```diff
@@ -261,14 +266,23 @@ At this point, with just the **serial_number** and **system** defined: our robot
...
+  - **ps4**: standard Playstation4 controller.
+  - **logitech**: Logitech F710
+
+```yaml
+controller: ps4 # or logitech
+```
+
 #### Attachments
 
 There are three types of attachments:
```
Comment (tonybaltovski) [inline docs/ros/config/yaml.mdx]: This says there are 3 types but the list below has 4 with two labelled as 2.  Should that be updated to be 4?

File: docs_versioned_docs/version-ros2humble/ros/config/live.mdx [hunk 2]
```diff
@@ -8,30 +8,46 @@ toc_max_heading_level: 4
...
 sudo apt install ros-humble-clearpath-desktop ros-humble-clearpath-config-live
 ```
 
 # Setup
+
 Before running the node, we will need to setup the working directory.
-First, create the directory `/etc/clearpath`.
-Then, copy your `robot.yaml` to `/etc/clearpath/robot.yaml`.
+
+Our standard working directory is: `/etc/clearpath`. However, you can choose whatever directory you'd like. We will refer to this directory as the **setup_path**.
```
Comment (hilary-luo) [inline docs_versioned_docs/version-ros2humble/ros/config/live.mdx]: `/etc/clearpath` is our standard working directory on the robot but in the "remote computer setup" (which we direct people to use for setting up their PC for simulation) we indicate `~/clearpath` as the standard. Maybe we should reference that here to say "Our standard working directory on robots is:  ... and our standard working directory on remote PC's and desktops is: ... or something similar. Or as an alternative below indicate that if they have setup their PC as per those instructions, the setup path is `$HOME/clearpath/`

**Review by hilary-luo — COMMENTED** [review]
As a note, it is unnecessary to make any changes in the `docs/` folder so don't worry in the future if it is out of date. It is not viewable on the site and whenever we choose to release a new version we will have to copy over the ros2 humble folder from the versioned section as a starting point (part of the process explained in the readme).

**Comment by hilary-luo** [comment]
There is mention of installing the micro ros workspace that needs to be removed here: https://docs.clearpathrobotics.com/docs/ros/installation/robot#source-install

**Review by hilary-luo — APPROVED** [review]

**Review by tonybaltovski — APPROVED** [review]

**Pull request merged**